driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -242,7 +245,14 @@ public void shouldPassAllOutcomes() {
         }
 
         if (definition.containsKey("expectLogMessages")) {
-            compareLogMessages(rootContext, definition);
+            ArrayList<LogMatcher.Tweak> tweaks = new ArrayList<>(singletonList(
+                    // `LogMessage.Entry.Name.OPERATION` is not supported, therefore we skip matching its value
+                    LogMatcher.Tweak.skip(LogMessage.Entry.Name.OPERATION)));
+            if (getMongoClientSettings().getClusterSettings()
+                    .getHosts().stream().anyMatch(serverAddress -> serverAddress instanceof UnixServerAddress)) {
+                tweaks.add(LogMatcher.Tweak.skip(LogMessage.Entry.Name.SERVER_PORT));
+            }
```

Comment:
This change is supposed to fix https://spruce.mongodb.com/task/mongo_java_driver_tests_socket__version~4.2_os~linux_topology~standalone_auth~auth_ssl~nossl_jdk~jdk8_socket_test_d375cd45b2cb60cf4400e8f0ed250ef933af9604_23_11_01_16_11_45/tests?execution=0&sortBy=STATUS&sortDir=ASC.